src/GraphBLAS-sharp.Backend/Algorithms/BFS.fs

```diff
@@ -67,14 +67,14 @@ module internal BFS =
             levels
 
     let singleSourceSparse
-        (add: Expr<bool option -> bool option -> bool option>)
-        (mul: Expr<bool option -> bool option -> bool option>)
+        (add: Expr<int option -> int option -> int option>)
```

Comment:
Why we should use ```int``` instead of ```bool```?

Reply:
The only reason is that spla and GraphBLAST use int and float matrices on BFS and comparison will be more fair. But now I think it is a strange reason so I'll change that to bool again